CONTRIBUTING.md

```diff
@@ -161,6 +161,42 @@ the docs will be built under `doc/_build/html`. Detail information about documen
 ## CI
 All PRs triger a CI job to run linting, type checking, tests, and build docs. The CI script is located [here](https://github.com/SeldonIO/alibi/blob/master/.github/workflows/ci.yml) and should be considered the source of truth for running the various development commands.
 
+## Optional Dependencies
```

Comment:
Do we need to update installation instructions for developers at the start of this doc to ensure all optional dependencies are installed too?

Reply:
I've changed the install option to `pip install -e .[all]`. I assume there isn't really a case where a developer doesn't want all the dependencies?